resources/wikia/Resources.php

```diff
@@ -183,6 +184,13 @@
 		],
 	],
 
+	'moment' => [
```

Comment:
Am I right if the 12k lines of the library will be loaded for every page view? Is it needed?